makefile

```diff
@@ -35,8 +35,18 @@ omim.owl: omim.ttl mondo_exactmatch_omim.sssom.owl mondo_exactmatch_omimps.sssom
 		query --update sparql/hgnc_links.ru \
 		convert -f ofn -o $@
 
-mondo_genes.csv: omim.owl
+mondo_genes.robot.tsv: omim.owl
+	# Create a TSV of relational information for gene and disease classes
 	robot query -i omim.owl --query sparql/mondo_genes.sparql $@
+	# Insert the source_code column as the second to last column
+	awk 'BEGIN {FS=OFS="\t"} {if (NR==1) {$$(NF+1)=$$(NF); $$(NF-1)="?source_code";} else {$$(NF+1)=$$(NF); $$(NF-1)="MONDO:OMIM";}} 1' $@ > temp_file && mv temp_file $@
+	# Remove the first character of each field in the header
+	awk 'BEGIN {FS=OFS="\t"} NR==1 {for (i=1; i<=NF; i++) $$i=substr($$i, 2)} {print}' $@ > temp_file && mv temp_file $@
+	# Remove < and > characters from specified columns
+	awk 'BEGIN {FS=OFS="\t"} NR>1 {gsub(/^<|>$$/, "", $$1); gsub(/^<|>$$/, "", $$2); gsub(/^<|>$$/, "", $$5)} {print}' $@ > temp_file && mv temp_file $@
```

Comment:
Update: Remove < and > characters from specified columns

@matentzn Do you know of an elegant, more standard way to do this?
Harshad and I use a common Python function sometimes to do this, and I think some of our libraries have functions for this as well, but I wanted to avoid Python here.

Reply:
Alrighty: 
- https://github.com/monarch-initiative/mondolib/issues/13